VTEX - Orders API.json

```diff
@@ -517,283 +423,50 @@
                                             }
                                         },
                                         "marketplace": {
-                                            "type": "object",
-                                            "description": "Details about the marketplace related to the order.",
-                                            "properties": {
-                                                "baseURL": {
-                                                    "type": "string",
-                                                    "description": "Marketplace base URL."
-                                                },
-                                                "isCertified": {
-                                                    "type": "boolean",
-                                                    "nullable": true,
-                                                    "description": "Defines if it is a [VTEX certified marketplace](https://help.vtex.com/en/tutorial/marketplace-strategies-at-vtex--tutorials_402#integrating-with-a-certified-marketplace) (`true`) or not (`false`)."
-                                                },
-                                                "name": {
-                                                    "type": "string",
-                                                    "description": "Marketplace name."
-                                                }
-                                            }
+                                            "$ref": "#/components/schemas/MarketplaceUpdated"
                                         },
                                         "storePreferencesData": {
+                                            "$ref": "#/components/schemas/StorePreferencesDataUpdated"
+                                        },
+                                        "customData": {
                                             "type": "object",
-                                            "description": "Store's configuration data, stored in [Account settings](https://help.vtex.com/en/tutorial/account-details-page--2vhUVOKfCaswqLguT2F9xq).",
+                                            "description": "Customizable fields created by the store for the shopping cart. This field is useful for storing data not included in other fields. For example, a message for a gift or a name to be printed in a shirt.",
```

Comment:
Is this related to Catalog attachments / custom product attributes? (asking beacuse of the examples)

If so, I suggest linking to related docs.